datafusion/common/src/file_options/parquet_writer.rs

```diff
@@ -156,6 +156,7 @@ pub(crate) fn parse_encoding_string(
         "plain" => Ok(parquet::basic::Encoding::PLAIN),
         "plain_dictionary" => Ok(parquet::basic::Encoding::PLAIN_DICTIONARY),
         "rle" => Ok(parquet::basic::Encoding::RLE),
+        #[allow(deprecated)]
```

Comment:
https://github.com/apache/arrow-rs/pull/5318

Reply:
I don't understand the reference (to the JSON writer) when this is for parquet encoding. Is there some other encoding/compression scheme that was deprecated too?

Reply:
This is a copypasta meant to link to https://github.com/apache/arrow-rs/pull/5348